src/components/ContentSidebar/Skills/__tests__/SidebarSkills-test.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import SidebarSection from '../../SidebarSection';
+import SidebarSkills from '../SidebarSkills';
+import * as skillUtils from '../skillUtils';
+
+describe('components/ContentSidebar/Skills/SidebarSkills', () => {
+    const getWrapper = (props) => mount(<SidebarSkills {...props} />);
```

Comment:
I'm wondering why `mount` is used here. The snapshot file is huge...

Reply:
@wenboyu2 the reason is mainly for the assertion

```
        expect(wrapper.find(SidebarSection)).toHaveLength(2);
```

shallow does not appear to render the components from the array.map. Do you have a better approach? Perhaps I can try stubbing the components out?

Reply:
Let me try recommitting when #252 is in